src/components/ProfilePage/DateJoined.jsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage, FormattedDate } from 'react-intl';
 
 function DateJoined({ date }) {
   if (date == null) return null;
 
-  const year = (new Date(date)).getFullYear();
-
   return (
-    <p className="mb-0">Member since {year}</p>
+    <FormattedMessage
+      id="profile.date.joined"
```

Comment:
The vague convention I've been using is reponame.componentname.something.about.the.message, so in this case profile.datejoined.member.since is probably what I'd call it.  That said, it actually makes no difference at all.